bin/incent

```diff
@@ -0,0 +1,450 @@
+
+Options:
+  General:
+    -h  Display this usage information
+
+  API:
+    -p  Program-Set ID
+    -e  Endpoint (e.g. 'calculations/bulk', 'calculations', 'incentives', 'qualification_results', 'calculations/stream')
+    -f  Path to request JSON
+
+  Metadata:
+    -p  Program-Set ID
+
+  API|Metadata|Token:
+    -E  Environment ('stage' or 'prod'; default 'stage')
+
+  Calculate:
+    -p  Program-Set ID
+    -f  Path to request JSON
+
+  Config:
+    init  Initialize a config file if one is not present
+    list  List current configuration settings
+
+  Local:
+    down  Tear down a currently running local instance of the incent-api
+    logs  Show the logs for the locally running instance
+    up    Run the incent-api locally in a container
+      -d  Run in a detached state
+
+Commands:
+  api            Send GET or POST requests to the Incent API (e.g. 'metadata/products', 'calculations/bulk')
+  config         Perform configuration functions given by sub-command (e.g. init, list)
+  calculate      Send a calculation request to the Incent API
+  local          Run an instance of the incent-api image locally
+  metadata       Generate a CSV report listing all configured program metadata in given program-set
+  token          Generate an Auth0 token
+
+Usage examples:
+  API:
+    Send GET or POST requests to the Incent API
+    Defaults to endpoint 'metadata/calculators' and enviroment 'stage':
+    > incent -p [program-set-id] api
+    # or
+    > incent -E prod -p [program-set-id] -e metadata/products api
+
+    Send POST request with request json:
+    > incent -E prod -p [program-set-id] -e calculations/bulk -f request.json api
+
+  Calculate:
+    Calculate request in staging:
+    > incent -p [program-set-id] -f request.json calculate
+    # or
+    > incent -E stage -p [program-set-id] -f request.json calculate
+
+  Config:
+    Establish a configuration file:
+    > incent config init
+    Print current configuration settings:
+    > incent config list
+
+  Local:
+    Run the incent-api locally in a container
+    > incent local up
+    Run in a detached state
+    > incent -d local up
+    Tear down a currently running local instance of the incent-api
+    > incent local down
+    View the logs
+    > incent local logs
+
+  Metadata:
+    Generate report in production:
+    > incent -E prod -p [program-set-id] metadata
+
+  Token:
+    Retrieve an Auth0 token for staging:
+    > incent token
+    Retrieve an Auth0 token for production:
+    > incent -E prod token"
+}
+
+secrets_not_available() {
+    echo "Secrets not found.
+
+  - have you configured incent? > incent config init
+"
+}
+
+token() {
+  local audience="${environment:-stage}"
+
+  if [ "${audience}" = "dev" ]; then
+    audience="stage"
+  fi
+
+  environment_key="${audience^^}"
+
+  if [[ "${audience}" = pr-* ]]; then
+    # review-apps use convention: tk-stage-incent-api-pr-559
+    app_suffix="-$audience"
+    audience="stage"
+    environment_key="REVIEW"
```

Comment:
I think it's safe to remove support review apps here. If we want to keep this then we'll need more Auth0 credentials (which can be problematic).